app/api/v1/auth/index.js

```diff
@@ -152,8 +190,7 @@ router.post('/resetPassword/:accessCode', (req, res, next) => {
 
 /**
  * Verifies a user's auth token, given the token in the 'Authorization' request header in the form of 'Bearer <TOKEN>'.
- * Responds with a JSON object with fields 'error', 'authenticated' (if a user's token is valid), and 'admin' (if the
- * user is an admin).
+ * Responds with a JSON object with fields 'error', 'authenticated' (if a user's token is valid)
```

Comment:
can you check if this was a change you made or another merge thing?

Reply:
My change, the comment was old and we only return authenticated now anyway